composer.json

```diff
@@ -24,7 +24,8 @@
   "require": {
     "yiisoft/yii2": "~2",
     "bazilio/yii2-async": "^0.2.0",
-    "yarcode/yii2-daemon": "^0.3"
+    "yarcode/yii2-daemon": "^0.3",
+    "yarcode/yii2-async-mailer": "*"
```

Comment:
self reference?

Reply:
sorry it's my first pull request 
the change I wanted to propose it's only the first one: added return statement in send methods